packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -217,18 +335,26 @@ harden(startPSM);
  * Make anchor issuer out of a Cosmos asset; presumably
  * USDC over IBC. Add it to BankManager.
  *
+ * Also, if vatParameters shows an anchorPoolBalance for this asset,
+ * mint a payment for that balance.
+ *
  * TODO: address redundancy with publishInterchainAssetFromBank
  *
- * @param {EconomyBootstrapPowers & WellKnownSpaces} powers
+ * @param {EconomyBootstrapPowers & WellKnownSpaces & ChainStorageVatParams} powers
  * @param {{options?: { anchorOptions?: AnchorOptions } }} [config]
  */
 export const makeAnchorAsset = async (
   {
-    consume: { agoricNamesAdmin, bankManager, zoe },
+    vatParameters: { chainStorageEntries },
+    consume: { agoricNamesAdmin, bankManager, zoe, anchorBalancePayments },
     installation: {
       consume: { mintHolder },
     },
-    produce: { testFirstAnchorKit },
+    // XXX: prune testFirstAnchorKit in favor of anchorMints
```

Comment:
what's `anchorMints`?

Reply:
it got replaced by `anchorBalancePayments`. good catch.